cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -416,3 +416,19 @@ def test_custom_button_ansible_credential_list(custom_service_button, service_ca
     credentials_dropdown = BootstrapSelect(appliance.browser.widgetastic, id="credential")
     all_options = [option.text for option in credentials_dropdown.all_options]
     assert ["<Default>", "CFME Default Credential"] == all_options
+
+
+def test_ansible_group_id_in_payload(service_catalog, service_request, service):
+    """Test if group id is presented in manageiq payload.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1480019
+    """
+    service_catalog.order()
+    service_request.wait_for_request()
+    view = navigate_to(service, "Details")
+    stdout = view.provisioning.standart_output
```

Comment:
Is standart_output misspelled?

Reply:
Definitely

Reply:
Cool glad I caught that.

Reply:
Misspell fix is out of the scope because it's a part of the view.

Reply:
:man_facepalming: